clientUtils/Bounds.ts

```diff
@@ -88,25 +88,23 @@ export class Bounds {
     ): Bounds {
         // Collapse contiguous spaces into one
         str = str.replace(/ +/g, " ")
-        const key = `${str}-${fontSize}`
-        let bounds = this.textBoundsCache[key]
-        if (bounds) {
-            if (bounds.x === x && bounds.y === y - bounds.height) return bounds
-            return bounds.extend({ x: x, y: y - bounds.height })
-        }
```

Comment:
I'm cool with removing the bounds cache from a correctness perspective. What exactly will it make slower though? The site baking?

Reply:
Have not tested, but my intuition is that the difference won't be measurable, or at least insignificant. The text bounds are calculated by concat-ing the bounds of every letter. The bounds of every letter are in a lookup table in the `string-pixel-width` library: https://github.com/adambisek/string-pixel-width/blob/master/src/widthsMap.js

Reply:
Yes I wrote some of the rationale in the commit message of the commit that does this but I realize now that this does not show up as context in the usual Github workflow - here is what I wrote then: 
    The method we use to calculate text bounds is already a fast heuristic and the memory overhead of keeping a bounds cache that includes the full string to be measured as a key is probably not worth it.
I checked that for the full svg generation this does not make a noticeable runtime difference. The memory use looked a bit lower in my totally non-scientific looking at the process explorer ;). At some point we should try to set up a memory benchmark for the grapher BTW